index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <title>Assignment-2</title>
+</head>
+<body>
+<!-- header start -->
+    <header>
+        <div class="logo"><h2>Team Shivam</h2></div>
+        <form action="" class="search">
+            <input type="search" name="" id="">
+        </form>
+    </header>
+<!-- navbar start -->
+    <nav>
+        <ul>
+            <li>Section 1</li>
+            <li>Section 2</li>
```

Comment:
add anchor tags instead of li tag to and provide route links in href value.
`<a class="options" href="section1">Section 1</a>`